ale/transformation.py

```diff
@@ -123,15 +124,20 @@ def from_spice(cls, sensor_frame, target_frame, center_ephemeris_time, ephemeris
 
         constant_frames.extend(target_constant_frames)
 
-        frame_chain.compute_time_dependent_rotiations(sensor_time_dependent_frames, sensor_times)
-        frame_chain.compute_time_dependent_rotiations(target_time_dependent_frames, target_times)
+        # Add all time dependent frame chains to the graph
```

Comment:
It seems that `use_web` is becoming ubiquitous in function signatures. Have other options been considered to not extend the signatures? For example, a module level variable or something?

Reply:
The function structure is definitely scuffed. Currently the `use_web` variable is in the NaifSpice mixin, which makes some sense as you want it on a driver to driver level.

Then we have the `kernel_access.py` module with the `spiceql_call` wrapper. This wrapper function is then further wrapped in the `NaifSpice` mixin where the `use_web` variable is grabbed from the driver when the function is called.

The issue here is that `transformation.py` and specifically the `FrameChain.from_spice` code is not part of the driver and exists to support getting the reference frame rotations. This requires that either a variable be passed through or that the FrameChain itself assigns it's own `use_web` variable which may actually be worth while.

So, to answer your question, I had not considered other options but I don't think a module level define is good. But I think we can make it that any FrameChain constructed using `from_spice` can set a member variable defining web usage.